pkg/resource/logging-secret/alloy/logging-secret.yaml.template

```diff
@@ -0,0 +1,45 @@
+# This file was generated by logging-operator.
+# It configures Alloy to be used as a logging agent.
+# - configMap is generated from logging.alloy.template and passed as a string
+#   here and will be created by Alloy's chart.
+# - Alloy runs as a deamonset, with required tolerations in order to scrape logs
+#   from every machine in the cluster.
+# - Running as root user is required in order to be able to read log files within
+#   /var/log/journal and /run/log/journal directories.
+# - NODENAME env var is used as additional label for kubernetes_audit logs.
+alloy-logs:
+  alloy:
+    configMap:
+      create: true
+      content: |-
+{{ .AlloyConfig | indent 8 }}
+    extraEnv:
```

Comment:
I feel like this as well as other properties should be part of the app itself and not this

Reply:
What do you mean ?

Reply:
I'm thinking most of what you put hère might bé more relevant as default in the alloy app than hère :)

Reply:
I added those values as example into the Alloy app https://github.com/giantswarm/alloy-app/pull/21

Reply:
Does this need to be a secret?

Reply:
Especially since you mentino in the comments that this is a configmap :D

Reply:
It's a configmap now